runtime: Deduplicate phrasing in state attribute definitions

wking opened the pull request:
Replace:

    $ENTRY ($TYPE) $ENTRY is the ...

with:

    $ENTRY ($TYPE) The ...

It's more concise and avoids the need to argue over Pid vs PID ;).

This is my first commit from crosbymichael/specs#1, rebased to pickup
the `version` addition that landed while #87 was in-flight.

laijs commented:
LGTM

philips commented:
I like the style that reads like a sentence. For example:

-* **root** (string) is the path to the container's bundle directory.

wking commented:
On Wed, Sep 09, 2015 at 09:47:49AM -0700, Brandon Philips wrote:
> I like the style that reads like a sentence. For example:
> 
> -* **root** (string) is the path to the container's bundle directory.		

Sounds good to me.  Fixed and rebased onto the current master with
81faf1e → 49f1bbd.

vbatts commented:
@wking looks like a funky rebase that is picking up other commits?

wking commented:
On Wed, Sep 09, 2015 at 07:56:17PM -0700, Vincent Batts wrote:
> @wking looks like a funky rebase that is picking up other commits?

I may have been bitten by the actions behind this IRC comment:

  14:48 < vbatts> i just used github's API to force push master from a
    commit that is only in their reflog

And here's the history I've seen for the master branch:

  $ git reflog origin/master
  9a8748c refs/remotes/origin/master@{0}: fetch --append origin: fast-forward
  fbac038 refs/remotes/origin/master@{1}: fetch --append origin: forced-update
  fd0ff56 refs/remotes/origin/master@{2}: fetch --append origin: fast-forward
  931c6a9 refs/remotes/origin/master@{3}: fetch --append origin: fast-forward
  …

Here's a graph of what I see:

  $ git log --graph --topo-order --oneline --decorate origin/master dedup-state-attribute-definitions fd0ff56
  *   9a8748c (origin/master, origin/HEAD) Merge pull request #160 from mrunalp/cap_fix
  |\  
  | * 663be9d (origin/pr/160) Modify the capabilities constants to match header files like other constants
  * |   6c4b37e Merge pull request #165 from wking/remove-mounts-destination-docs
  |\ \  
  | * | 713d981 (wking/remove-mounts-destination-docs, origin/pr/165, remove-mounts-destination-docs) runtime-config: Remove 'destination' docs from mounts
  | |/  
  | | *   fd0ff56 (master) Merge pull request #163 from laijs/RootfsPropagation
  | | |\  
  | | | * 339e038 (origin/pr/163) Deduplicate the field of RootfsPropagation
  | | |/  
  | |/|   
  | | | * 49f1bbd (HEAD, wking/dedup-state-attribute-definitions, origin/pr/145, dedup-state-attribute-definitions) runtime: Deduplicate phrasing in state attribute definitions
  | | |/  
  | | *   931c6a9 Merge pull request #159 from mrunalp/rlimit_type
  | | |\  
  | |/ /  
  |/| |   
  | | * d3a6069 (origin/pr/159) Change the rlimit type to string instead of int
  | |/  
  * |   fbac038 Merge pull request #164 from philips/add-devices-to-linux-spec
  |\ \  
  | |/  
  |/|   
  | * 3848a23 (origin/pr/164) config-linux: specify the default devices/filesystems available
  |/  
  *   6cc6798 Merge pull request #162 from vbatts/formatting
  …

So it looks like this PR, the #163 merge, and the #159 merge were
abandoned by your force-push.  I haven't gone through the IRC history
in more detail yet to understand what you were trying to accomplish,
but I'll report back once I have a clearer idea.

wking commented:
On Wed, Sep 09, 2015 at 08:25:32PM -0700, W. Trevor King wrote:
> So it looks like this PR, the #163 merge, and the #159 merge were
> abandoned by your force-push.  I haven't gone through the IRC history
> in more detail yet to understand what you were trying to accomplish,
> but I'll report back once I have a clearer idea.

I haven't been able to figure out what was wrong from the IRC logs.
Here are the entries that appeared relevant:

14:23 < vbatts> crosbymichael: what commit is your origin/master at
14:23 < vbatts> before git pull
14:25 < vbatts> i think it was force pushed
14:26 < vbatts> :-|
14:27 < crosbymichael> umm
14:27 < crosbymichael> 16147
14:27 < crosbymichael> 527a3ee
14:27 < crosbymichael> that
14:27 < crosbymichael> now 26b9be2
14:28 < vbatts> it should be fbac038b38ef0cc8bef62c911e658f41f98ff42d
14:28 < vbatts> https://github.com/opencontainers/specs/commit/fbac038b38ef0cc8bef62c911e658f41f98ff42d
14:29 < vbatts> https://github.com/opencontainers/specs/pulls?q=is%3Apr+is%3Aclosed it's missing the most recent 7 merges
14:48 < vbatts> i just used github's API to force push master from a commit that is only in their reflog
14:49 < vbatts> tianon: `curl -u vbatts -H "Authorization: token FARTFARTFARTS" --request PATCH https://api.github.com/repos/opencontainers/specs/git/refs/heads/master
                --data '{"sha": "fbac038b38ef0cc8bef62c911e658f41f98ff42d", "force": true}'`
14:49 < vbatts> crosbymichael: origin is fixed now

So I'm not sure how you decided on fbac038b3, but I think you should
at least merge fd0ff56c8e into the current master.  And folks should
stop force-pushing to master ;).

wking commented:
On Wed, Sep 09, 2015 at 07:56:17PM -0700, Vincent Batts wrote:
> @wking looks like a funky rebase that is picking up other commits?

With 3f62423 and 712a746 you pulled in #159 and #163.  Those were the
only PRs that were part of the abandoned fd0ff56 history, so I've
rebased my commit onto origin/master and we're back down to just my
change.

vbatts commented:
LGTM

wking commented:
This has been rolled into #211.
